.travis.yml

```diff
@@ -5,8 +5,6 @@ cache:
 language: java
 jdk:
   - oraclejdk8
-services:
-  - redis-server
```

Comment:
Why do we remove this one?
I can guess because all the metrics stuff has been moved to the separate project, but at least some comment in the commit would shed light for "Why?"
Thanks

Reply:
@artembilan Updated the commit with some additional changes and description. The `codecov` report is now available as well.